src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: "What's new in Astro - October"
+description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
+publishDate: "October 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-october/head.webp"
+socialImage: "/src/content/blog/_images/whats-new-october/og.webp"
+lang: "en"
+---
+
+## Astro releases
```

Comment:
Yeah I saw that — do we generally not want to mention Starlight releases? I was suggesting including them here rather than having a whole extra section, but also OK if we’ve decided they shouldn’t be mentioned.

Reply:
That was me! I was OK mentioning releases of all kinds, BUT... then do we have to do the cloudflare adapter, the mdx integration etc?

I'm OK adding Starlight back in here, and with renaming to releases, but keeping it to Astro meant we didn't have to make those choices yet.  What do y'all think?  :sweat_smile: